environment

```diff
@@ -74,6 +74,10 @@ DSS_SUBSCRIPTION_AUTHORIZED_DOMAINS=${DSS_SUBSCRIPTION_AUTHORIZED_DOMAINS_ARRAY[
 DSS_SUBSCRIPTION_AUTHORIZED_DOMAINS_TEST="human-cell-atlas-travis-test.iam.gserviceaccount.com"
 DSS_ES_DOMAIN="dss-index-$DSS_DEPLOYMENT_STAGE"
 PYTHONWARNINGS=ignore:ResourceWarning,ignore::UserWarning:zipfile:
+DSS_PARAMETER_STORE="/dss/parameters"
```

Comment:
please include an explanation of what these new environment variables are for in the readme.